package.json

```diff
@@ -2,23 +2,25 @@
   "name": "monsters-rolodex",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://nishantc7.github.io/simple-pokedex",
```

Comment:
Change this back to the previous homepage and script. You can do this change on your forked repo.